src/truncated/normal.jl

```diff
@@ -27,56 +27,59 @@ modes(d::Truncated{Normal{T},Continuous}) where {T <: Real} = [mode(d)]
 # do not export. Used in mean
 # computes mean of standard normal distribution truncated to [a, b]
 function _tnmom1(a, b)
+    mid = middle(a, b)
```

Comment:
I guess we should use
```suggestion
    mid = float(middle(a, b))
```
since `middle` does not necessarily return a floating point number:
```julia
julia> StatsBase.middle(0//1, 1//1)
1//2
```

Reply:
Ah, good call, thanks!